Dockerfile

```diff
@@ -1,12 +1,12 @@
 # Keep hub.Dockerfile aligned to this file as far as possible
 ARG JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
-ARG APP_INSIGHTS_AGENT_VERSION=2.3.1-SNAPSHOT
+ARG APP_INSIGHTS_AGENT_VERSION=2.3.1
```

Comment:
this will remove your instrumentation of database calls, this was built specifically for CCD...
You should instead upgrade to version 2.5.0 of app insights, requires java-logging upgrade, note that the artifact ids have dropped java in them now